src/__tests__/InputField.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react'
+import renderer from 'react-test-renderer'
+import 'jest-styled-components'
+import { ThemeProvider, InputField } from '..'
+
+describe('InputField', () => {
+  test('it renders', () => {
+    const json = renderer.create(
+      <ThemeProvider>
```

Comment:
We can actually use `import { theme } from '..'` here and pass `theme` as a prop instead of using `ThemeProvider`. I start seeing people using different patterns for how to test their components. I used to it the way you did it and then I have changed my mind after I had a discussion with @jxnblk. `ThemeProvider` adds an extra dependency for testing the components, while if by using `theme={theme}`, `theme` is static and removes some overhead. I am also looking for a solution on wrapping programmatically all the components tested in Jest with a ThemeProvider.

Reply:
Since InputField has children that are dependent on the theme being in context, I need to use ThemeProvider in this case :(

Reply:
To add some "context" to this discussion 😉  https://github.com/facebook/react/issues/7905